### PR TITLE
URL apostrophes bug fix

### PR DESCRIPTION
**Pull request by dansterenson**
## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: https://github.com/demisto/etc/issues/36751

## Description
Added support for apostrophes when extracting URL.

### CODE REVIEW

**Review by teizenman — CHANGES_REQUESTED** [review]
Add a test to the playbook

**Comment by reutshal** [comment]
> Add a test to the playbook

@dansterenson There is a test-playbook that checks the indicators extraction, called - "Indicators-reputation-Test"
Please add an example of the URL and make sure we extract it as expected.

File: Packs/Base/TestPlaybooks/playbook-Indicators-reputation-.json-Test.yml [hunk 1]
```diff
@@ -1017,6 +1017,14 @@ tasks:
...
             value:
               simple: http://www.foo.bar/taz?())
+      - - operator: notContainsGeneral
+          left:
+            value:
+              simple: ${URL.Data}
+            iscontext: true
+          right:
+            value:
+              simple: https://Test.com/this'that
```
Comment (teizenman) [inline Packs/Base/TestPlaybooks/playbook-Indicators-reputation-.json-Test.yml:1027]: It's not the same as the input, is that on purpose?

**Review by reutshal — APPROVED** [review]

File: Packs/Base/TestPlaybooks/playbook-Indicators-reputation-.json-Test.yml (hunk 1 above)
Comment (dansterenson) [reply Packs/Base/TestPlaybooks/playbook-Indicators-reputation-.json-Test.yml:1027]: Yes, when I upload the updated playbook and execute it, I get:

Before the changes (URL doesn't support apostrophes):
image

After the changes (URL does support apostrophes):
image

**Review by teizenman — APPROVED** [review]

**Pull request merged**